source/reference/methods.txt

```diff
@@ -376,6 +381,13 @@ Connection Methods
 
      - Sets the :term:`read preference` for the MongoDB connection.
 
+   * - :method:`Mongo.watch()`
```

Comment:
Is there documentation anywhere for the ChangeStreamCursor? The majority of the backwards breaking changes regarding the watch method / change streams are in the cursor API.

Reply:
@aherlihy looks like the change stream cursor documentation is in the server manual with the [db.collection.watch()](https://docs.mongodb.com/manual/reference/method/db.collection.watch/#db.collection.watch) method.

Reply:
Ok, as long as somewhere is updated to reflect those updates LGTM